### DISCUSSION TITLE
Best practise for integrating django-hijack

### DISCUSSION THREAD

**Discussion by dbartenstein** [farridav/django-jazzmin#259] [Q&A] [answered]
First of all - thanks a lot for this awesome project! :clap:

I am using django hijack and would also like to display the hijack notification in the admin interface:
https://django-hijack.readthedocs.io/en/stable/#after-installing

Unfortunately there is no appropriate block defined after the body tag, so I temporarily put it into the extrahead block, which of course make the HTML invalid. On the other hand I don’t want to duplicate jazzmin’s base.html

Do you have an idea on how to best tackle this? Thanks a lot!

```
{% extends "admin/base.html" %}
{% load static hijack_tags %}

{% block extrastyle %}
<link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack-styles.css' %}" />
{% endblock %}

{% block extrahead %}
{% comment %}
This is misplaced here as it belongs to the beginning of the body.
Unfortunately there no is block available at the beginning of the body.
Copying the whole body.html would not be nice.
{% endcomment %}
{% hijack_notification %}
{% endblock %}
```

**Comment by dbartenstein** [comment, marked as answer]
I might have found a viable solution by putting hijack_notification into the page_content block:
```
{% extends "admin/base.html" %}
{% load static hijack_tags %}

{% block extrastyle %}
<link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack-styles.css' %}" />
{% endblock %}

{% block page_content %}
{% hijack_notification %}
{{ block.super }}
{% endblock %}
```

**Reply by farridav** [reply]
Have you tried putting it above messages ? thats probably the right place for it ..

```
{% extends "admin/base.html" %}
{% load static hijack_tags %}

{% block extrastyle %}
<link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack-styles.css' %}" />
{% endblock %}

{% block messages %}
{% hijack_notification %}
{{ block.super }}
{% endblock %}
```
?

**Reply by dbartenstein** [reply]
Hi @farridav - thanks for your suggestion! That’s also a possible approach. It depends where one wants to display the hijack notification - below or above the breadcrumbs.

**Comment by PacificGilly** [comment]
Marking discussion as closed as a workaround has been given. Feel free to reopen/comment if you have any more to discuss on this 🙂
